### DISCUSSION TITLE
Close #222

### DISCUSSION THREAD

**Discussion by iamapig120** [one-among-us/data#224] [Q&A]
已收到相对可靠消息据信转生，希望close issue

老实说，我觉得用死亡这件事来转生/神隐真的很可恶，浪费人对他人的关心。

有些难受

**Comment by Twinsherry** [comment]
在近期发生的一些事件背景下，我们曾对这类问题有所讨论，但没有确定性的结果，也不公开发表意见，但我们会尽可能维护这片园地的纪念严肃性。
